apps/box/src/components/Cards/ConnectedDevicesCard.tsx

```diff
@@ -43,31 +40,33 @@ const DeviceCard = ({
       delayLongPress={200}
     >
       <FxText color="content1" variant="bodyLargeRegular" marginBottom="8">
-        {name}
+        {data.name}
```

Comment:
could de-structure keys from data as they are used in multiple places: 

`const { name, capacity, status } = data`

Reply:
done